go.mod

```diff
@@ -15,7 +15,7 @@
 
 module github.com/apache/spark-connect-go/v1
 
-go 1.19
+go 1.21
```

Comment:
This is not strictly necessary for the code itself but some of the `error.Is` functionality will not work in 1.19. It will compile fine and work with whatever features existed in 1.19 though

Reply:
Good catch. Fixed